### DISCUSSION TITLE
0.3.1-rc04

### DISCUSSION THREAD

**Discussion by gary-hammer** [bellrichm/weewx-jas#91] [General]
I installed 0.3.1-rc04 this morning, deleting the output directory tree after install.
Initial run is fine and we will have high winds and plenty of rain this afternoon.

Anything to look for?

**Comment by bellrichm** [comment]
First, in the ‘min/max’ section change windSpeed to wind (or add wind).
Second, when you see bad values capture your site. You can try these [directions](https://github.com/bellrichm/weewx-jas/wiki/Debugging#debugging-display-issues). At the very least capture the data/last24hours.js (assuming that is the ‘page’ being displayed)
Thanks!

**Comment by gary-hammer** [comment]
Change made.
So, for rain to show the accumulating total from MQTT for current conditions does this look correct?

[[[[[[rain]]]]]]
     type = sum
     mqtt = false

**Reply by bellrichm** [reply]
Sorta’….  weewx-jas doesn’t do summation of MQTT data. So that will get the sum for the ‘page’ time period and update the display on the archive period.
I’m not going to add that complexity into weewx-jas. It is up to the source to publish the MQTT data that is desired to be displayed.

**Reply by gary-hammer** [reply]
So, if I just want what is in the MQTT field, remove the 'sum'?

**Reply by bellrichm** [reply]
Remove both.

**Reply by gary-hammer** [reply]
Perfect.

**Reply by gary-hammer** [reply]
This is what I now have.
`[[[[[[rain]]]]]]
      max_decimals = 2
 [[[[[[rainRate]]]]]]`

Working well. Wind is correct as well.

**Reply by bellrichm** [reply]
That’s great. Thanks for testing and letting me know!
